docs/contributing/contributing-documentation.md

```diff
@@ -0,0 +1,137 @@
+
+```bash
+$ git clone https://github.com/YOUR-GITHUB-USERNAME/netdata.git
+```
+
+You can now jump into the directory and explore Netdata's structure for yourself.
+
+
+### Understanding the structure of Netdata's documentation
+
+All of Netdata's documentation is stored within the repository itself, as close as possible to the code it corresponds to. Many sub-folders contain a `README.md` file, which is then used to populate the documentation about that feature/component of Netdata.
+
+For example, the file at `packaging/installer/README.md` becomes `https://docs.netdata.cloud/packaging/installer/` and is our installation documentation. By co-locating it with quick-start installtion code, we ensure documentation is always tightly knit with the functions it describes.
+
+You might find other `.md` files within these directories. The `packaging/installer/` folder also contains `UPDATE.md` and `UNINSTALL.md`, which become `https://docs.netdata.cloud/packaging/installer/update/` and `https://docs.netdata.cloud/packaging/installer/uninstall/`, respectively.
+
+If the documentation you're working on has a direct correlation to some component of Netdata, place it into the correct folder and either name it `README.md` for generic documentation, or with another name for very specific instructions.
+
+#### The `docs` folder
+
+At the root of the Netdata repository is a `docs/` folder. Inside this folder we place documentation that does not have a direct relationship to a specific component of Netdata. It's where we house our [getting started guide](../GettingStarted.md), guides on [running Netdata behind Nginx](../Running-behind-nginx.md), and more.
+
+If the documentation you're working on doesn't have a direct relaionship to a component of Netdata, it can be placed in this `docs/` folder.
+
+
+### Make your edits
+
+Now that you're set up and understand where to find or create your `.md` file, you can now begin to make your edits. Just use your favorite editor and keep in mind our [style guide](style-guide.md) as you work.
+
+Be sure to periodically add/commit your edits so that you don't lose your work! We use version control software for a reason.
+
+
+### Build the documentation
+
+Building the documentation periodically gives you a glimpse into the final product, and is generally required if you're making changes to the table of contents.
+
+!!! attention ""
+    The build process has only been tested on Linux, but it should also work on OS X machines. Windows is fully untested at this point, but we would love to know if it works there as well!
+
+To build the documentation, you need `python`/`pip`, `mkdocs`, and `mkdocs-material` installed on your machine.
+
+Follow the [Python installation instructions](https://www.python.org/downloads/) for your machine.
+
+Use `pip`, which was installed alongside Python, to install `mkdocs` and `mkdocs-material`.
```

Comment:
It is necessary to take with command "pip", some distributions use "pip" for python 2.x, while they have "pip3" for python 3.x .